explore/src/main/scala/explore/config/package.scala

```diff
@@ -50,3 +62,95 @@ trait ConfigurationFormats:
         case WavelengthUnits.Nanometers  => wvDeltaNanoInput
 
 object ConfigurationFormats extends ConfigurationFormats
+
+case class ModeData private (
```

Comment:
This was moved from AdvancedConfigurationPanel and modified a bit. There, it had `centralWavelength` and `interval` parameters. But, it seems the algorithm for finding the ModeData was changed to use the spectroscopy requirements wavelength instead of central wavelength, so those parameters were no longer valid nor used.